src/data/ability.ts

```diff
@@ -5779,9 +5779,14 @@ export function initAbilities() {
       .attr(WonderSkinAbAttr)
       .ignorable(),
     new Ability(Abilities.ANALYTIC, 5)
-      .attr(MovePowerBoostAbAttr, (user, target, move) =>
-        !!target?.getLastXMoves(1).find(m => m.turn === target?.scene.currentBattle.turn)
-        || user?.scene.currentBattle.turnCommands[target?.getBattlerIndex() ?? BattlerIndex.ATTACKER]?.command !== Command.FIGHT, 1.3),
+      .attr(MovePowerBoostAbAttr, (user, target, move) => {
+        const movePhase = user?.scene.findPhase((phase) => phase instanceof MovePhase && phase.pokemon.id !== user.id);
+        if (movePhase) {
+          return false;
+        } else {
+          return true;
+        }
```

Comment:
```suggestion
        return Utils.isNullOrUndefined(movePhase);
```
Just a style change (this file still has the `Utils` import pikaStare)

Reply:
Ah, good point.

`Utils.thingsandstuff` should be gone soonish... just need to update my import reform PR.